### PR TITLE
Added windows openssl script

### PR DESCRIPTION
**Pull request by christianpaquin**
Added PowerShell script to test Windows's OpenSSL build.

### CODE REVIEW

**Comment by christianpaquin** [comment]
The diff shows mods to `integration/oqs_openssl/run.sh`. I haven't touched that file, I don't know if my merge reverted some things. @dstebila, @smashra, can you make sure that file is the latest? (I made a local diff across branches and it looked ok)

File: integration/oqs_openssl/run.sh [hunk 1]
```diff
@@ -52,31 +52,46 @@ echo "Cloning openssl"
...
-rm -rf "${BASEDIR}/openssl-1_0_2-master"
-cp -pr "${BASEDIR}/openssl" "${BASEDIR}/openssl-1_0_2-master"
-cd "${BASEDIR}/openssl-1_0_2-master"
+if [ ! -d "${BASEDIR}/openssl_1_1_1-stable" ] ; then
+	git clone -b OpenSSL_1_1_1-stable https://github.com/openssl/openssl.git "${BASEDIR}/openssl_1_1_1-stable" >> $LOGS 2>&1 
+fi
+
+cd "${BASEDIR}/openssl_1_1_1-stable"
+case "$OSTYPE" in
+  darwin*) CFLAGS=-fPIC  ./Configure shared --prefix="${BASEDIR}/install-openssl_1_1_1-stable" darwin64-x86_64-cc >> $LOGS 2>&1 ;;
```
Comment (dstebila) [inline integration/oqs_openssl/run.sh]: The change from `./Configure no-shared` to `CFLAGS=-fPIC ./Configure shared` is causing a bunch of problems on my Mac.  Is there a reason you made this change?

Comment (christianpaquin) [reply integration/oqs_openssl/run.sh]: No, I think it's due to an out-of-sync commit. As noted in my comment above; some changes was applied to run.sh (I'm not sure why and how), but I've only added the windows test file.

File: integration/oqs_openssl/README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
 OQS-OpenSSL Integration Testing
 ===============================
 
-This directory contains a script for testing the OQS forks of OpenSSL with liboqs.
+This directory contains scripts for testing the OQS forks of OpenSSL with liboqs, using all supported algorithms. The README.md files for the OQS-OpenSSL forks ([1.0.2](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_0_2-stable/README.md), [1.1.1](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_1_1-stable/README.md)) describe the various key exchange and authentication mechanisms supported by each configuration.
+
+The script `run.sh` runs on Linux and macOS, and the PowerShell script `winrun.ps1` runs on Windows. First make sure you have **installed the dependencies** for the target OS as indicated in the [top-level testing README](https://github.com/open-quantum-safe/testing/blob/master/README.md).
```
Comment (dstebila) [inline integration/oqs_openssl/README.md]: @christianpaquin The top-level testing README doesn't containing anything about dependencies for Windows.  Does anything need to be added there?  Or does it run without any special requirements?

Comment (christianpaquin) [reply integration/oqs_openssl/README.md]: I guess so; I've created issue #13 to remember.

**Pull request merged**